packages/slice-machine/src/components/SideNav/SideNav.css.ts

```diff
@@ -316,7 +316,7 @@ export const rightElementText = style([
     alignSelf: "baseline",
     fontSize: "12px",
     lineHeight: "16px",
-    maxWidth: "68px",
+    maxWidth: "82px",
```

Comment:
That's pretty big right?

Reply:
It's for the text "Login required" so the ellipsis don't trigger.
I tested it for the version ad it works well with our alpha one